### DISCUSSION TITLE
support for .p12/.pfx files for mTLS

### DISCUSSION THREAD

**Discussion by cplussharp** [panva/openid-client#326] [Ideas]
Hi,
I would like to use a .p12 file for mTLS.
This is currently not possible, because the code checks for "key" and "certificate" in the https options.
If they are not present the code should check for "pfx".

**Comment by panva** [comment]
Sounds good. Would you like to open a PR?

**Reply by cplussharp** [reply]
Yes, I can do this. I never worked with discussions before. Can I reference this discussion like an issue?

**Reply by panva** [reply]
Same as any other issue. #{number}
